ui/v2.5/src/models/list-filter/scenes.ts

```diff
@@ -39,6 +39,7 @@ const sortByOptions = [
   "interactive",
   "interactive_speed",
   "perceptual_similarity",
+  "code",
```

Comment:
I think this should be `scene_code`.

Reply:
The problem with ListFilterOptions is that it'll [set the value as messageID](https://github.com/stashapp/stash/blob/develop/ui/v2.5/src/models/list-filter/filter-options.ts#L25). So it will either display the correct messageID but fail the query as the database field is called `code` and not `scene_code` or it will display the value because `code` is not a valid messageID. With my very limited React knowledge I couldn't find a solution to this so I decided to remove the sort by scene code option entirely in 66043ff.